src/Makefile.am

```diff
@@ -129,6 +129,6 @@ endif
 
 libgdiplus_la_LIBADD = $(GDIPLUS_LIBS)
 
-AM_CPPFLAGS = $(GDIPLUS_CFLAGS) -Wall -Wno-unused -Wno-format
+AM_CPPFLAGS = $(GDIPLUS_CFLAGS)
```

Comment:
This seems to trigger the build errors, btw.

Reply:
The 'gdip_setpixel_32bppARGB' function is really unused. Somehow `-Wunused` is apparently already included in the warnings and this removed `-Wno-unused` then triggers the build error.